.eslintrc

```diff
@@ -12,6 +12,7 @@
   "rules": {
     "no-console": "off",
     "strict": ["error", "global"],
-    "curly": "warn"
+    "curly": "warn",
+    "getter-return": "error"
```

Comment:
Isn't this our own `eslintrc`? To update the rules of our config, you'd need to change `eslint-config-react-app/index.js`

Reply:
Ah good catch, I'll update the config

Reply:
Is it worth keeping this new rule in the root eslint config for development work?